app/scripts/views/components/barChart/barChartTemplate.js

```diff
@@ -201,9 +202,16 @@
       if (((this.data.max - this.data.min) > 1000) && (this.data.min > 1)) {
         this.settings.showLogScale = true;
       }
-      this.initChart(this.settings.showLogScale);
-      this.updateShowSurvivalIcon();
-      this.$dispatch('data-loaded', this.attributes.group_id, this.chartDivId);
+      
+      if (isNaN(this.data.min) || isNaN(this.data.max)) {
```

Comment:
Is there a possible for this condition. I think all `NaN` are filtered in while finding extremes(`var findExtremeResult = cbio.util.findExtremes(this.data.meta);`). If its a `NaN` shoudn't it throw an error at line 202?